src/bucket/BucketManagerImpl.cpp

```diff
@@ -1521,4 +1523,72 @@ BucketManagerImpl::getConfig() const
 {
     return mApp.getConfig();
 }
+
+std::string
+to_string(LedgerEntryTypeAndDurability const type)
```

Comment:
Nit: `toString`. Also this would be a good utility function to move to the `LedgerTypeUtils` file as well.